src/triangle.h

```diff
@@ -13,7 +13,7 @@
 typedef struct Triangle {
     ShapeOps* op;
     Shader* shader;
-    Vec3  point[1];
+    Vec3  point[3];
```

Comment:
#1 This should be changed back and use point[0] + edge[n] to get the original points.